tensorflow/contrib/opt/python/training/elastic_average_optimizer.py

```diff
@@ -139,6 +186,8 @@ def __init__(self,
       rho: the amount of exploration we allow ine the model. The default
         value is moving_rate/learning_rate
       use_locking: If True use locks for update operations.
+      sync_flag: Add_sync_queues_and_barrier or not, default to False, in case of
```

Comment:
Explain what sync_queues_and_barrier is, and when people should set this to true or false

Reply:
True: all workers will wait for each other before start training
False: worker can start training when its initilization is done, no need to wait for everyone is ready.
           in case one worker is restarted, it can join and continue training without being blocked.

Will update these in to code.